terraform/environment/ecs_front_v2.tf

```diff
@@ -0,0 +1,211 @@
+//----------------------------------
+// front v2 ECS Service level config
+
+resource "aws_ecs_service" "front-v2" {
+  name                  = "front-v2"
+  cluster               = aws_ecs_cluster.online-lpa.id
+  task_definition       = aws_ecs_task_definition.front.arn
```

Comment:
this needs to be changed to v2 reference

Reply:
Thanks. Sorted :)